src/components/past-events/past-events.jsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import classes from './past-events.less';
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4069302/22962995/708fff9a-f361-11e6-8abc-97a0cf5b73d0.png)
I think it's more convenient to store colors in less, and it could be globally used on the pages